client/src/components/App.jsx

```diff
@@ -26,10 +27,22 @@ class App extends React.Component {
     .catch(() => {console.log("Error: Reviews not found.")}
```

Comment:
not clear why you're trying to catch an error here.

Reply:
In case the promise doesn't fetch anything. Do you suggest something else?